### PR TITLE
[ADD] DCOS-2269 add docs to powershell script

### PR DESCRIPTION
**Pull request by YuraTolochkevich**

### CODE REVIEW

File: win_bin/install/install-dcos-cli.ps1 [hunk 1]
```diff
@@ -95,3 +95,8 @@ dcos config set package.cache $env:temp\dcos\package-cache
 dcos config set package.sources '[\"https://github.com/mesosphere/universe/archive/version-1.x.zip\"]'
 
 dcos package update
+
+echo "Finished installing and configuring DCOS CLI."
+echo ""
+echo "Run this command to set up your environment and to get started:"
+echo "<install_dir>\Scripts\activate.ps1"
```
Comment (jsancio) [inline win_bin/install/install-dcos-cli.ps1]: Please give a better message. We know the `install_dir` it should be the env variable `$installation_path`, no?

**Comment by YuraTolochkevich** [comment]
@jsancio 
I added `$dcos_installation_path` variable.

**Comment by jsancio** [comment]
> @jsancio I added $dcos_installation_path variable.

Thanks. Can you please copy and paste what this script prints when executed?

**Comment by YuraTolochkevich** [comment]
PS C:\Users\Administrator\Downloads> .\install.ps1 c:\dsd36 localhost
True
Installing DCOS CLI from PyPI...



    Directory: C:\


Mode                LastWriteTime     Length Name
----                -------------     ------ ----
d----         7/14/2015   7:52 PM            dsd36
New python executable in c:\dsd36\Scripts\python.exe
Installing setuptools, pip, wheel...done.
Updating source [https://github.com/mesosphere/universe/archive/version-1.x.zip]
Finished installing and configuring DCOS CLI.

Run this command to set up your environment and to get started:
& "$env:dcos_installation_path\Scripts\activate.ps1"

PS C:\Users\Administrator\Downloads> & "$env:dcos_installation_path\Scripts\activate.ps1"
(dsd36) PS C:\Users\Administrator\Downloads> deactivate

**Pull request merged**